terms.html

```diff
@@ -5,12 +5,20 @@
 <dl class="termlist">
   <dt><dfn data-lt="claims">claim</dfn></dt>
   <dd>
-An assertion made about a <a>subject</a>.
+A digital representation of an assertion made, by a specific <a>entity</a>, about a <a>resource</a>.
+This <a>resource</a> is called the <a>subject</a> of the <a>claim</a>.
+This digital representation usually consists of an (optional) identifier (an [IRI](https://datatracker.ietf.org/doc/html/rfc3987))
+that identifies this <a>subject</a>, and one or more attributes or properties (key-value pairs, or predicate-object graphs).
+The meaning of a claim (its semantics) is determined by its author.
+Claims can be authored by <a>issuers</a>, e.g., to construct a <a>verifiable credential</a> with, or 
+by <a>holders</a>, e.g., to include in a <a>verifiable presentation</a>.
+Different <a>issuers</a> and <a>verifiers</a> can reuse the same semantics for claims through the use of shared vocabularies and/or ontologies.
```

Comment:
We have taken a one liner and expanded it into a, while accurate statement, something that is nearly impossible to keep in one's head and recite to another person. The new text is far more accurate, and nearly impossible to convey easily in a conversation. One approach we could take here is to move all the precise text into another section and refer to that section from the pithy definition.

Reply:
Happy to comply. Any suggestions where to put that text, and which parts should be moved?